blogs/how-to-get-involved-to-rocky-linux-and-get-some-resources.md

```diff
@@ -28,7 +28,7 @@ tags:
 
 ## 台灣社群
 
-* [台灣 Rocky Linux 網站](https://rockylinux.tw/)：就是本網站，歡迎任何人參與貢獻，可以參考[貢獻方法](https://github.com/rockylinuxtw/rockylinux.tw/CONTRIBUTING.md)。
+* [台灣 Rocky Linux 網站](https://rockylinux.tw/)：就是本網站，歡迎任何人參與貢獻，可以參考[貢獻方法](https://rockylinux.tw/CONTRIBUTING.html)。
```

Comment:
Suggest using https://github.com/rockylinuxtw/rockylinux.tw/blob/develop/CONTRIBUTING.md.

Reply:
ok, change original to this one.